src/ntfs/read.rs

```diff
@@ -327,6 +324,12 @@ pub fn manage_comments(collections: &mut Collections, path: &path::Path) -> Resu
 
         let path = path.join("comment_links.txt");
         if let Ok(mut rdr) = csv::Reader::from_path(&path) {
+            // invert the stop_time_ids map to search a stop_time by it's id
+            let stop_time_ids: HashMap<String, (Idx<VehicleJourney>, u32)> = collections
+                .stop_time_ids
+                .iter()
+                .map(|(k, v)| (v.clone(), *k))
+                .collect();
```

Comment:
with 
```rust
let stop_time_ids: HashMap<&String, (Idx<VehicleJourney>, u32)> = collections
                .stop_time_ids
                .iter()
                .map(|(k, v)| (v, *k))
                .collect();
```
you can improve memory
on fr-idf, before 4,42GB, after: 3,74GB

Reply:
```rust
HashMap<&str, (Idx<VehicleJourney>, u32)>
```

Using `BTreeMap` or a `flat_map::FlatMap` can also improve memory.

Reply:
`&str` add useless complexity, so my comment is fixed.